### PR TITLE
fix: call still ringing despite that the caller already ended the call (AR-3046) (RC)

### PR DESCRIPTION
**Pull request by ohassine**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-3046" title="AR-3046" target="_blank">BugAR-3046</a>  Applause - [SG A52s/A.13] 1:1 Chat - Call still ringing despite that the caller already ended the call
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Same PR as #1575 to be merged into RC branch

### Causes (Optional)

_Briefly describe the causes behind the issues. This could be helpful to understand the adopted solutions behind some nasty bugs or complex issues._

### Solutions

_Briefly describe the solutions you have implemented for the issues explained above._

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1592/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.0(6102999)(6102999)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.0(6102999)(6102999)-staging-compat.apk)

**Review by alexandreferris — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1592/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.0(6105275)(6105275)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.0(6105275)(6105275)-staging-compat.apk)

**Review by trOnk12 — APPROVED** [review]

File: buildSrc/src/main/kotlin/Dependencies.kt [hunk 1]
```diff
@@ -80,7 +80,7 @@ object Libraries {
         const val pinEditText = "1.2.3"
         const val sqlLiteJdbc = "3.36.0"
         const val desugaring = "1.1.5"
-        const val workManager = "2.7.1"
```
Comment (tmspzz) [inline buildSrc/src/main/kotlin/Dependencies.kt]: Nope, this updates a library

**Review by tmspzz — DISMISSED** [review]

File: buildSrc/src/main/kotlin/Dependencies.kt (hunk 1 above)
Comment (ohassine) [reply buildSrc/src/main/kotlin/Dependencies.kt]: reverted

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-1592/5/) **failed**.

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-1592/6/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.0(6112596)(6112596)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.0(6112596)(6112596)-staging-compat.apk)

**Review by yamilmedina — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [7](https://10.10.124.134/job/wire-android-reloaded/job/PR-1592/7/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.0(6130144)(6130144)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.0(6130144)(6130144)-staging-compat.apk)

**Review by tmspzz — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [8](https://10.10.124.134/job/wire-android-reloaded/job/PR-1592/8/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.1.0(6172461)(6172461)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.1.0(6172461)(6172461)-staging-compat.apk)

**Pull request merged**